src/installer/tests/HostActivation.Tests/NativeHosting/FunctionPointerLoading.cs

```diff
@@ -0,0 +1,219 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Cli.Build.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation.NativeHosting
+{
+    public partial class FunctionPointerLoading : IClassFixture<ComponentActivation.SharedTestState>
```

Comment:
I would:
* Rename the existing `ComponentActivation` test to `LoadAssemblyAndGetFunctionPointer`
* Rename this to `GetFunctionPointer`

To make it much more obvious which test belongs to what runtime functionality.